baseplate/context/thrift.py

```diff
@@ -52,25 +65,48 @@ def _enumerate_service_methods(client):
 class PooledClientProxy(object):
     """A proxy which acts like a thrift client but uses a connection pool."""
 
-    def __init__(self, client_cls, pool, root_span, namespace):
+    def __init__(self, client_cls, pool, root_span, namespace, retry_policy=None):
```

Comment:
Oh.  I read that as "1 retry".  Well that explains it then.  PEBKAC.  I'm content with the current behavior.